src/main/java/be/polyscripts/contactmanagerapp/model/Contact.java

```diff
@@ -0,0 +1,47 @@
+package be.polyscripts.contactmanagerapp.model;
+
+import lombok.*;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
+
+@Builder
+@AllArgsConstructor @NoArgsConstructor @Getter @Setter
+@Entity
+@Table(name = "contact")
+public class Contact {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id", nullable = false)

```

Comment:
make sue to not expose your PKs via you REST services

Reply:
done !